test/ia-menu-slider.test.js

```diff
@@ -4,7 +4,7 @@ import { IAMenuSlider } from '../src/ia-menu-slider.js';
 customElements.define('ia-menu-slider', IAMenuSlider);
 
 const container = (menus = []) => (
-  html`<ia-menu-slider .menus=${menus}></ia-menu-slider>`
+  html`<ia-menu-slider .menus=${menus} @ItemNavMenuClosed=${() => {}}></ia-menu-slider>`
```

Comment:
QUESTION: Did you want to be testing `<ia-menu-slider>` with its Boolean `open` attribute set in some way too?

Reply:
Good question. We are actually testing the `open` boolean in various unit tests using the class/element instance.  so we are covered in accessing and changing that property.

I've actually updated the demo to account for toggling `this.open` from the outside.

This is actually an artifact from devving, as well.  yanked.